discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/model/UpdateConfigurationOptions.java

```diff
@@ -205,18 +195,18 @@ public Builder enrichments(List<Enrichment> enrichments) {
      * @return the UpdateConfigurationOptions builder
      */
     public Builder configuration(Configuration configuration) {
-      this.conversions = configuration.getConversions();
-      this.name = configuration.getName();
-      this.description = configuration.getDescription();
-      this.normalizations = configuration.getNormalizations();
-      this.enrichments = configuration.getEnrichments();
+    this.conversions = configuration.getConversions();
+    this.name = configuration.getName();
+    this.description = configuration.getDescription();
+    this.normalizations = configuration.getNormalizations();
+    this.enrichments = configuration.getEnrichments();
```

Comment:
More funky formatting here.  This case looks like some spacing issues in the template.  But odd that it is showing up here and not in the other classes.

Reply:
Oh ... I bet this is one of the "alternate-setters".  Not in the template ... these lines are produced in the generator itself.